packages/python/test/fixtures/14-unicode-handler/probes.json

```diff
@@ -0,0 +1,16 @@
+{
+  "probes": [
+    {
+      "path": "/",
+      "method": "POST",
+      "body": {
+        "name": "Χριστοφορε"
+      },
+      "status": 200,
+      "mustContain": "hello, \\u03a7\\u03c1\\u03b9\\u03c3\\u03c4\\u03bf\\u03c6\\u03bf\\u03c1\\u03b5",
```

Comment:
This looks different than the original test

Reply:
This is mostly testing what the original test was testing. It tests that the response is json and the content includes `hello, Χριστοφορε`.

Original: 

```
if (respBody.greeting !== 'hello, Χριστοφορε') {
  throw new Error(`unexpected response: ${respBody}`);
}
```

Reply:
In the original test, it was unescaped via `JSON.parse`. The fetch and response are otherwise identical between old and new versions of this test.

```
> JSON.parse('{"greeting": "hello, \\u03a7\\u03c1\\u03b9\\u03c3\\u03c4\\u03bf\\u03c6\\u03bf\\u03c1\\u03b5"}');
{ greeting: 'hello, Χριστοφορε' }
```

I tried a few things around unescaping this from any `testDeployment`, but there wasn't a clear path to that. I'd be happy to pair on this if you think it's still an issue so I can better understand what's happening here. Otherwise, I do think the new test asserts the same thing as the old one, essentially.